hydromt_fiat/fiat.py

```diff
@@ -733,6 +736,29 @@ def setup_aggregation_areas(
         self.exposure.exposure_db = join_exposure_aggregation_areas(
             exposure_gdf, aggregation_area_fn, attribute_names, label_names
         )
+        
+        # Create additional attributes folder in output
+        additional_att = Path(self.root).joinpath("output" , "additional_attributes")
+        if not os.path.exists(additional_att):
+            os.makedirs(additional_att)
+
+        if isinstance(aggregation_area_fn,list):
+            for file in aggregation_area_fn:
+                shutil.copy2(file, additional_att)
+        else:
+            shutil.copy2(aggregation_area_fn, additional_att)
```

Comment:
```suggestion
```
Just saving it in the additional_attributes folder in the root folder is sufficient!